Makefile

```diff
@@ -41,10 +41,11 @@ clean:
 
 setup-git:
 	@echo "--> Installing git hooks"
-	pip install "pre-commit>=1.10.1,<1.11.0"
-	pre-commit install
 	git config branch.autosetuprebase always
+	git config core.ignorecase false
```

Comment:
🤔 Isn't this bad for case insensitive filesystems like macOS's default? Or do I have this backwards of when you'd want this set or not.

Reply:
@mattrobenolt its actually important because macOS is case sensitive (linux is not). In the past we had mistakes in the getsentry repo due to this (and had been running it there for a while).

Reply:
You have it backwards, macOS is insensitive and Linux is not (generally). And yeah, this definitely causes us issues since on Mac, you can't even rename the file, haha

Reply:
err yeah, so its more just making sure we dont fuck it up because of differences